<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color spaces</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QcvMatWidget" name="widgetImage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>463</width>
         <height>534</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="sizeTab">
       <attribute name="title">
        <string>Size</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioButtonOrigSize">
          <property name="text">
           <string>Originale</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonCustomSize">
          <property name="text">
           <string>Custom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelWidth">
          <property name="text">
           <string>Largeur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxWidth">
          <property name="maximum">
           <number>1600</number>
          </property>
          <property name="singleStep">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelHeight">
          <property name="text">
           <string>Hauteur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxHeight">
          <property name="maximum">
           <number>1200</number>
          </property>
          <property name="singleStep">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxFlip">
          <property name="text">
           <string>Flip</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>228</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="colorTab">
       <attribute name="title">
        <string>Color</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <item>
         <widget class="QGroupBox" name="displayBox">
          <property name="title">
           <string>Display</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>8</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>8</number>
           </property>
           <property name="bottomMargin">
            <number>8</number>
           </property>
           <item>
            <widget class="QRadioButton" name="radioButtonInput">
             <property name="toolTip">
              <string>Input Image (I)</string>
             </property>
             <property name="text">
              <string>Input</string>
             </property>
             <property name="shortcut">
              <string extracomment="I">I</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonGray">
             <property name="toolTip">
              <string>Grayscale Image (K)</string>
             </property>
             <property name="text">
              <string>Gray</string>
             </property>
             <property name="shortcut">
              <string>K</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonRed">
             <property name="toolTip">
              <string>Red Component (R)</string>
             </property>
             <property name="text">
              <string>RGB: Red</string>
             </property>
             <property name="shortcut">
              <string>R</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonGreen">
             <property name="toolTip">
              <string>Green Component (G)</string>
             </property>
             <property name="text">
              <string>RGB: Green</string>
             </property>
             <property name="shortcut">
              <string>G</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonBlue">
             <property name="toolTip">
              <string>Blue Component (B)</string>
             </property>
             <property name="text">
              <string>RGB: Blue</string>
             </property>
             <property name="shortcut">
              <string>B</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonMaxBGR">
             <property name="toolTip">
              <string>RGB Maximum Image</string>
             </property>
             <property name="text">
              <string>RGB: Max</string>
             </property>
             <property name="shortcut">
              <string>M</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonXYZ_X">
             <property name="toolTip">
              <string>X Component (X)</string>
             </property>
             <property name="text">
              <string>XYZ: X</string>
             </property>
             <property name="shortcut">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonXYZ_Y">
             <property name="toolTip">
              <string>Y component (W)</string>
             </property>
             <property name="text">
              <string>XYZ: Y</string>
             </property>
             <property name="shortcut">
              <string>W</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonXYZ_Z">
             <property name="toolTip">
              <string>Z Component (Z)</string>
             </property>
             <property name="text">
              <string>XYZ: Z</string>
             </property>
             <property name="shortcut">
              <string>Z</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonHue">
             <property name="toolTip">
              <string>Hue Component (H)</string>
             </property>
             <property name="text">
              <string>HSV: Hue</string>
             </property>
             <property name="shortcut">
              <string>H</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonSaturation">
             <property name="toolTip">
              <string>Saturation Component (S)</string>
             </property>
             <property name="text">
              <string>HSV: Saturation</string>
             </property>
             <property name="shortcut">
              <string>S</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonValue">
             <property name="toolTip">
              <string>Value Component (V)</string>
             </property>
             <property name="text">
              <string>HSV: Value</string>
             </property>
             <property name="shortcut">
              <string>V</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonY">
             <property name="toolTip">
              <string>Y Component (Y)</string>
             </property>
             <property name="text">
              <string>YCbCr: Y</string>
             </property>
             <property name="shortcut">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonCr">
             <property name="toolTip">
              <string>Green/Magenta Chrominance (U)</string>
             </property>
             <property name="text">
              <string>YCbCr: Cr</string>
             </property>
             <property name="shortcut">
              <string>U</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonCb">
             <property name="toolTip">
              <string>Blue/Yellow Chrominance (T)</string>
             </property>
             <property name="text">
              <string>YCbCr: Cb</string>
             </property>
             <property name="shortcut">
              <string>T</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="channelBox">
          <property name="title">
           <string>Channel</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>8</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>8</number>
           </property>
           <property name="bottomMargin">
            <number>8</number>
           </property>
           <item>
            <widget class="QRadioButton" name="radioButtonChColor">
             <property name="text">
              <string>Color</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonChGray">
             <property name="text">
              <string>Gray</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="hueBox">
          <property name="title">
           <string>Hue mix</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>8</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>8</number>
           </property>
           <property name="bottomMargin">
            <number>8</number>
           </property>
           <item>
            <widget class="QRadioButton" name="radioButtonMixHue">
             <property name="text">
              <string>Hue</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonMixHueSat">
             <property name="text">
              <string>Hue x Sat</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonMixHueVal">
             <property name="text">
              <string>Hue x Value</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="processTimeLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="labelProcessTime">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>process time:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelProcessTimeValue">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelPM">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>±</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelProcessTimeStd">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelTimeUnit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>ms</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSources">
    <property name="title">
     <string>Sources</string>
    </property>
    <addaction name="actionCamera_0"/>
    <addaction name="actionCamera_1"/>
    <addaction name="actionFile"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Video</string>
    </property>
    <widget class="QMenu" name="menuSize">
     <property name="title">
      <string>taille</string>
     </property>
     <addaction name="actionOriginalSize"/>
     <addaction name="actionConstrainedSize"/>
    </widget>
    <addaction name="actionFlip"/>
    <addaction name="separator"/>
    <addaction name="menuSize"/>
   </widget>
   <widget class="QMenu" name="menuRender">
    <property name="title">
     <string>Render</string>
    </property>
    <addaction name="actionRenderImage"/>
    <addaction name="actionRenderPixmap"/>
    <addaction name="actionRenderOpenGL"/>
   </widget>
   <addaction name="menuSources"/>
   <addaction name="menuVideo"/>
   <addaction name="menuRender"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCamera_0">
   <property name="text">
    <string>Camera 0</string>
   </property>
   <property name="toolTip">
    <string>caméra interne</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+0</string>
   </property>
  </action>
  <action name="actionCamera_1">
   <property name="text">
    <string>Camera 1</string>
   </property>
   <property name="toolTip">
    <string>caméra externe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+1</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
   <property name="toolTip">
    <string>fichier video</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFlip">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>flip</string>
   </property>
  </action>
  <action name="actionOriginalSize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Originale</string>
   </property>
   <property name="toolTip">
    <string>taille originale</string>
   </property>
  </action>
  <action name="actionConstrainedSize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Custom</string>
   </property>
   <property name="iconText">
    <string>contrainte</string>
   </property>
   <property name="toolTip">
    <string>taille imposée</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="toolTip">
    <string>quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRenderImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionRenderPixmap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pixmap</string>
   </property>
  </action>
  <action name="actionRenderOpenGL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OpenGL</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QcvMatWidget</class>
   <extends>QWidget</extends>
   <header>QcvMatWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonCustomSize</sender>
   <signal>clicked(bool)</signal>
   <receiver>actionConstrainedSize</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConstrainedSize</sender>
   <signal>triggered(bool)</signal>
   <receiver>radioButtonCustomSize</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonOrigSize</sender>
   <signal>clicked(bool)</signal>
   <receiver>actionOriginalSize</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOriginalSize</sender>
   <signal>triggered(bool)</signal>
   <receiver>radioButtonOrigSize</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxFlip</sender>
   <signal>clicked(bool)</signal>
   <receiver>actionFlip</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFlip</sender>
   <signal>triggered(bool)</signal>
   <receiver>checkBoxFlip</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
